scripts/userstyles.yml

```diff
@@ -416,6 +416,13 @@ userstyles:
     readme:
       app-link: "https://news.ycombinator.com"
     current-maintainers: [*lucasmelin]
+  haveibeenpwned:
```

Comment:
Probably should do proper kebab case right?
```suggestion
  have-i-been-pwned:
```

Reply:
i suppose

Reply:
Make sure to update the folder name, userstyles.yml metadata, and usercss metadata.